databroker/broker.py

```diff
@@ -217,10 +221,10 @@ def __call__(self, text_search=None, **kwargs):
 
         Examples
         --------
-        >>> DataBroker('keyword')  # full text search
-        >>> DataBroker(start_time='2015-03-05', stop_time='2015-03-10')
-        >>> DataBroker(data_key='motor1')
-        >>> DataBroker(data_key='motor1', start_time='2015-03-05')
+        >>> Broker('keyword')  # full text search
```

Comment:
Probably should not use `Broker` here is this needs to be an instance of this class, not the class it's self.